docs/getting-started-guides/docker.md

```diff
@@ -75,12 +74,11 @@ docker run \
     --privileged=true \
     -d \
     gcr.io/google_containers/hyperkube:v${K8S_VERSION} \
-    /hyperkube kubelet \
-        --containerized \
+    /kubelet-runner.sh \
         --hostname-override="127.0.0.1" \
         --address="0.0.0.0" \
         --api-servers=http://localhost:8080 \
-        --config=/etc/kubernetes/manifests \
+        --config=etc/kubernetes/manifests \
         --cluster-dns=10.0.0.10 \
         --cluster-domain=cluster.local \
         --allow-privileged=true --v=10
```

Comment:
Could you decrease to `--v=3` or `--v=5`?
When 10, it is sometimes hard to debug